frontend/src/components/UserPage/ServiceRequests/PendingPaymentServiceRequests/PendingPaymentServiceRequests.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent, useCallback, useContext, useEffect, useState } from 'react';
+import { UserContext } from '../../../../contexts/User/UserContext';
+import { ServiceRequestDto } from '../../../../dtos/ServiceRequestDto';
+import { specialistsService } from '../../../../services/specialistsService';
+import { toast } from 'react-toastify';
+import { List } from 'semantic-ui-react';
+import { usersService } from '../../../../services/usersService';
+
+export interface Props {}
+
+export const PendingPaymentServiceRequests: FunctionComponent<Props> = () => {
+    const { user } = useContext(UserContext);
+    const [serviceRequests, setServiceRequests] = useState<ServiceRequestDto[]>([]);
+
+    const getServiceRequests = useCallback(() => {
+        if (user === null) {
+            throw new Error('User is null');
+        }
+        usersService
+            .getServiceRequests(user, 'FINISHED', false)
```

Comment:
We can create an enum with the service statuses since they are a small, discrete number.